setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(name='pyeafe',
+      version='0.1',
+      description='Edge-Averaged Finite Elements (EAFE) for FENiCS',
+      url='https://github.com/thepnpsolver/pyeafe',
+      author='Maximilien Metti, Shuonan Wu, Arthur Bousquet',
+      author_email='',
+      license='GNU General Public License v3.0',
```

Comment:
We could do MIT... or Apache.  We don't want to impose any restrictions on usage : )

Reply:
If we depend on fenics, then I think we aren't allowed to leave the GPL framework either...